src/app/features/public-form/edit-public-form-columns/edit-public-form-columns.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="hint-banner" i18n>
```

Comment:
This still needs a margin at the bottom:
![image](https://github.com/user-attachments/assets/7bc4d662-a237-4a46-95f1-3edbb762a496)